docs/section21/Rule21-11.md

```diff
@@ -0,0 +1,38 @@
+
+# Boiler - Rule 21-11  
+
+**Rule ID:** 21-11  
+**Rule Description:** When the system uses boilers the hot water system shall be modeled as primary only with continuous variable flow and a minimum turndown of 25%.  
+**Rule Assertion:** B-RMR  
```

Comment:
@mkarpman and @xingcx not sure if it makes sense to split this into 2 rules, one for continuous VF and if flow = VF, then minimum turndown should be = 25%. I guess its ok to leave it as  1 rules as these rules are dependent on each other.